sidechain/rpc-handler/src/import_block_api.rs

```diff
@@ -47,9 +47,17 @@ where
 
 		info!("sidechain_importBlock. Blocks: {:?}", blocks);
 
-		let _ = import_fn(blocks).map_err(|e| {
-			jsonrpc_core::error::Error::invalid_params_with_details("Failed to import Block.", e)
-		})?;
+		// FIXME: Error handling? For now only logging the error.
+		// FIMXE: why is it import_block and not import_blocks?
```

Comment:
@clangenb I wasn't sure what your initial intention was so for now I settled adaption of the `ImportFn` instead of making `import_block` to `import_blocks`. Does it matter for you where the `  for loop / mapping` of the multiple blocks happens?